distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -86,3 +86,4 @@ log4j.logger.org.apache.solr = WARN
 log4j.logger.lux.solr = WARN
 log4j.logger.org.ops4j.pax.web.jsp = WARN
 log4j.logger.org.apache.aries.spifly = WARN
+log4j.logger.org.apache.cxf.phase.PhaseInterceptorChain = ERROR
```

Comment:
Sets default logging level for PhaseInterceptorChain to ERROR instead of WARN. Can be set back with the `set:log level package` command.